app/styles/components/CardList.scss

```diff
@@ -45,7 +45,6 @@
       width: 100%;
       height: 219px;
       position: relative;
-      background-color: #D8D8D8;
```

Comment:
I think we should keep the same color and change it when we get to release 0.0.6